src/tests/classic/run.rs

```diff
@@ -807,3 +807,36 @@ fn test_modern_sets_source_file_in_symbols() {
         Some("resources/tests/steprun/fact.cl".to_string())
     );
 }
+
+// Test that leaving off the lambda captures causes bare words for the
+// requested values to find their way into the output and that having
+// the capture catches it.  This shows that uses of uncaptured words
+// are unencumbered.
```

Comment:
perhaps I misunderstand what this means, but I would hope that the compiler would catch my mistake if I misspell a variable name in the lambda body. having an explicit capture seems like an opportunity for us to be stricter about this

Reply:
As we discussed earlier, that wasn't a change I felt i could make at this point.  Since it's been very contentious I will speed through the changes needed to be fully strict.

Reply:
This upcoming pr allows us to get to strictness in macro output since it allows precise control of the form the output takes, as well as other improvements.

https://github.com/Chia-Network/clvm_tools_rs/pull/134

It's a precondition for being fully strict.